src/pybind/mgr/dashboard/services/proto/gateway_pb2.py

```diff
@@ -0,0 +1,3265 @@
+# -*- coding: utf-8 -*-
+# Generated by the protocol buffer compiler.  DO NOT EDIT!
+# source: gateway.proto
+"""Generated protocol buffer code."""
+from google.protobuf.internal import enum_type_wrapper
+from google.protobuf import descriptor as _descriptor
+from google.protobuf import message as _message
+from google.protobuf import reflection as _reflection
+from google.protobuf import symbol_database as _symbol_database
+# @@protoc_insertion_point(imports)
+
+_sym_db = _symbol_database.Default()
```

Comment:
We shouldn't version these 2 pb2 files. That's how it's done in the nvmeof project too. We should add a make or tox command to do so.